components/eventing-controller/Dockerfile

```diff
@@ -20,6 +20,6 @@ LABEL source = git@github.com:kyma-project/kyma.git
 WORKDIR /
 COPY --from=builder /go/src/github.com/kyma-project/kyma/components/eventing-controller/eventing-controller .
 COPY licenses/ /licenses/
-USER nonroot:nonroot
+USER 1001
```

Comment:
Can you provide some context why this change is required?

Reply:
Removed the change in Dockerfile